crates/matrix-sdk-indexeddb/src/cryptostore.rs

```diff
@@ -1051,3 +1090,18 @@ mod tests {
     }
     cryptostore_integration_tests! { integration }
 }
+
+#[cfg(test)]
+mod encrypted_tests {
+    use super::IndexeddbStore;
+    wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
```

Comment:
Why is this interleaved with imports? I think it should come after if there's no specific reason for having it in the middle here.

Reply:
I remember vaguely, that it needs to happen before the following import, but I am not sure this is still the case...